gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds
+from ...data import DataStore, ObservationList
+
+
+@requires_data('gammapy-extra')
+def test_make_psftable():
```

Comment:
> ok but I mean shouldn't we try to undersand why the HAP gaussian psf are so different for these two runs?

Yes, we should!
(but I could do that and that doesn't have to hold up this pull request)

Do you have an example code snippet I can run to start investigating (that e.g. prints some numbers or makes a plot, e.g. in the `examples` folder in this branch, or in a separate https://gist.github.com/) or should I write my own?